src/DataLoaders.py

```diff
@@ -0,0 +1,102 @@
+        else:
+            structure_set = set(line.strip() for line in open(include))
+        for id in tqdm(structure_set):
+            PDBid, chain = id.split(".")
+            if(PDBid in database):
+                structure = database[PDBid]["structure"]
+                protein_chain = structure[(structure.chain_id == chain) & struc.filter_amino_acids(structure)] 
+                if(check_chain(protein_chain)):
+                    self.protein_labels = pd.concat([self.protein_labels,  labels[labels["chain_id"] == id]])
+        
+    def __len__(self):
+        return len(self.protein_labels)
+
+    def __getitem__(self, idx):
+        labeled = self.protein_labels.iloc[idx]
+        PDBid, chain = labeled.iloc[0]["chain_id"].split(".")
+        x = dp.DataPreProcessorForGNM(type_flexibility = self.flex)
+        structure = self.database[PDBid]["structure"]
+        protein_chain = structure[(structure.chain_id == chain) & struc.filter_amino_acids(structure)] 
+        struct = [x.from_loaded_structure(protein_chain, m) for m in labeled["label"]]
+        return struct
+
+class AllProteinDataset(Dataset):
+    """
+    Dataset class for loading all proteins for available labels.
+    """
+    def __init__(self, labels, database, flex: Literal["msqf", "bfact", "pseudo"] = "msqf", exclude: Optional[str] = None):
+        self.flex = flex
+        self.protein_labels = pd.DataFrame(columns = ['chain_id', 'label'])
+        for id in tqdm(set(labels.chain_id).difference(set(line.strip().replace("_",".") for line in open(exclude)))):
+            PDBid, chain = id.split(".")
+            if(PDBid in database):
+                structure = database[PDBid]["structure"]
+                protein_chain = structure[(structure.chain_id == chain) & struc.filter_amino_acids(structure)] 
+                if(check_chain(protein_chain)):
+                    self.protein_labels = pd.concat([self.protein_labels,labels[labels["chain_id"] == id]])
+    
+    def __len__(self):
+        return len(self.protein_labels)
+
+    def __getitem__(self, idx):
+        labeled = self.protein_labels.iloc[idx]
+        PDBid, chain = labeled.iloc[0]["chain_id"].split(".")
+        x = dp.DataPreProcessorForGNM(type_flexibility = self.flex)
+        structure = self.database[PDBid]["structure"]
+        protein_chain = structure[(structure.chain_id == chain) & struc.filter_amino_acids(structure)] 
+        struct = [x.from_loaded_structure(protein_chain, m) for m in labeled["label"]]
+        return struct
+
+
+def load_labels(path:str):
+    """
+    Function for loadning labels from the file.
+
+    Arguments:
+        path: str - path to labels data
+    Return:
+        labels: pd.DataFrame - class labels for proteins
+    """
+    labels = pd.read_csv(path, names = ["chain_id", "label"])
+    return labels
+
+def check_chain(chain: biotite.structure.AtomArray):
```

Comment:
```suggestion
def is_chain_valid(chain: biotite.structure.AtomArray) -> bool:
```

Naming it `is_chain_valid` makes it clear that the output of this function will be a bool